src/js/model.js

```diff
@@ -1,24 +1,37 @@
-import { createMockServer } from "./Mock-Server.js";
+import { createMockServer } from "./mockServer.js";
 import { showSnackbar } from "./helper-functions.js";
-import { todoActions } from "./consts.js";
+import {
+  todoActions,
+  filterNames,
+  urgency,
+  category,
+  INVALID_POSITION,
+} from "./constants.js";
 
 export class Model {
   constructor() {
     this.allTodos = [];
     this.currentlySelected = [];
-    this.counter = 0;
     this.filterData = {
-      urgencyFilterMask: Array(3).fill(false), // true/false
-      categoryFilterMask: Array(3).fill(false),
+      urgencyFilterMask: {
+        [urgency.LOW]: false,
```

Comment:
don't need to explicitly initialise the mask as false. If it is not present in the object, it automatically equates to a `falsy` value